snap/build-scripts/build

```diff
@@ -27,7 +27,7 @@ if [ -z "$KUBE_SNAP_BINS" ]; then
             curl -LO \
               https://dl.k8s.io/${KUBE_VERSION}/bin/linux/$arch/$app
             chmod +x $app
-            if ! file ${app} 2>&1 | grep -q 'executable'; then
+            if [[ ! -x $app ]]; then
```

Comment:
This case will never occur 'cause it's checking for the executable bit, which we just set with `chmod +x` on the previous line.

The `file | grep executable` check has some history to it. I don't have time to pull up the history right this sec but I'm pretty sure we accidentally pushed out a snap whose "executable binary" was an XML file. We should be careful not to lose this if it's still protecting us from something.